be/src/Infrastructure/Localization/Foundation.Localization/Resources/OrderResources.de.resx

```diff
@@ -291,25 +291,49 @@
   <data name="oc_orderedProducts" xml:space="preserve">
     <value>Bestellte Produkte</value>
   </data>
-  <data name="oc_name" xml:space="preserve">
+  <data name="sh_nameLabel" xml:space="preserve">
     <value>Name</value>
   </data>
   <data name="oc_text" xml:space="preserve">
     <value>Wir freuen uns, Ihnen mitteilen zu können, dass Ihre Bestellung bestätigt wurde.</value>
   </data>
-  <data name="oc_outletQuantity" xml:space="preserve">
```

Comment:
**question:** Is this property used somewhere? because I'm afraid it is and we'll have to modify the appropriate template

Reply:
I only changed the names in the resource files; the template variables remain the same as before these changes.